src/goto-symex/symex_main.cpp

```diff
@@ -14,6 +14,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <cassert>
 #include <memory>
 
+#include <pointer-analysis/value_set_dereference.h>
```

Comment:
Random place: the commit message ("f filter value sets - use build_reference_to") needs some editing.